server/inspect_test.go

```diff
@@ -1,220 +1,129 @@
-package server
+package server_test
 
 import (
-	"testing"
-	"time"
+	"net/http"
+	"net/http/httptest"
 
-	runtime "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
-
-	"github.com/cri-o/cri-o/lib"
-	"github.com/cri-o/cri-o/lib/sandbox"
-	"github.com/cri-o/cri-o/oci"
-	specs "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/cri-o/cri-o/pkg/storage"
+	"github.com/go-zoo/bone"
+	"github.com/golang/mock/gomock"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
 )
 
-func TestGetInfo(t *testing.T) {
```

Comment:
What happened to these tests?

Reply:
Yeah I see we should keep them, I will leave them side-by-side to the new tests.